app/stores/ada/LedgerConnectStore.js

```diff
@@ -0,0 +1,389 @@
+      //     localizableError = new LocalizableError(globalMessages.trezorError102);
+      //     break;
+      //   case 'Cancelled':
+      //   case 'Popup closed':
+      //     localizableError = new LocalizableError(globalMessages.trezorError103);
+      //     break;
+      //   case 'Signing cancelled':
+      //     localizableError = new LocalizableError(messages.signTxError101);
+      //     break;
+      //   default:
+      //     /** we are not able to figure out why Error is thrown
+      //       * make it, Something unexpected happened */
+      //     Logger.error(`TrezorSendStore::_convertToLocalizableError::error: ${error.message}`);
+      //     localizableError = new UnexpectedError();
+      //     break;
+      // }
+    }
+
+    if (!localizableError) {
+      /** we are not able to figure out why Error is thrown
+        * make it, Something unexpected happened */
+      localizableError = new UnexpectedError();
+    }
+
+    return localizableError;
+  }
+
+  @action _goToConnectError = (): void => {
+    this.progressInfo.currentStep = ProgressStep.CONNECT;
+    this.progressInfo.stepState = StepState.ERROR;
+  };
+  // =================== CONNECT =================== //
+
+  // =================== SAVE =================== //
+  @action _goToSaveLoad = (): void => {
+    this.error = null;
+    this.progressInfo.currentStep = ProgressStep.SAVE;
+    this.progressInfo.stepState = StepState.LOAD;
+  };
+
+  /** SAVE dialog submit (Save button) */
+  @action _submitSave = (walletName: string): void => {
+    this.error = null;
+    this.progressInfo.currentStep = ProgressStep.SAVE;
+    this.progressInfo.stepState = StepState.PROCESS;
+    this._saveHW(walletName);
+  };
+
+  /** creates new wallet and loads it */
+  _saveHW = async (walletName: string): Promise<void>  => {
+    try {
+      Logger.debug('LedgerConnectStore::_saveHW:: called');
+      this._setIsCreateHWActive(true);
+      this.createHWRequest.reset();
+
+      const reqParams = this._prepareCreateHWReqParams(walletName);
+      const ledgerWallet: CreateHardwareWalletResponse =
+        await this.createHWRequest.execute(reqParams).promise;
+
+      if (ledgerWallet) {
+        await this._onSaveSucess(ledgerWallet);
+      } else {
+        // this Error will be converted to LocalizableError()
+        throw new Error();
+      }
+    } catch (error) {
+      Logger.error(`LedgerConnectStore::_saveHW::error ${stringifyError(error)}`);
+
+      if (error instanceof CheckAdressesInUseApiError) {
+        // redirecting CheckAdressesInUseApiError -> hwConnectDialogSaveError101
+        // because for user hwConnectDialogSaveError101 is more meaningful in this context
+        this.error = new LocalizableError(globalMessages.hwConnectDialogSaveError101);
+      } else if (error instanceof LocalizableError) {
+        this.error = error;
+      } else {
+        // some unknow error
+        this.error = new UnexpectedError();
```

Comment:
that is not swallow, it has fixed message inside it.
`UnexpectedError` is a `LocalizableError` for UI display (in case we are not sure which error happened)

debugging is done in upper block.
`Logger.error(`LedgerConnectStore::_saveHW::error ${stringifyError(error)}`);`